packages/cloudflare/src/cli/build/patches/plugins/build-id.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Inline `getBuildId` as it relies on `readFileSync`that is not supported by workerd.
```

Comment:
```suggestion
 * Inline `getBuildId` as it relies on `readFileSync` that is not supported by workerd.
```

Reply:
fixed (and the source as it was a copy/paste)